admin/availability/middleware.js

```diff
@@ -29,4 +40,20 @@ async function isAdminWindowAvailable (req, res, next) {
   }
 }
 
+/**
+   * The UK academic year starts in September and continues through to August of the following year.
+   * TODO: move to lib
+   * @param {moment.Moment} date
+   * @return {number}
+   */
+function getAcademicYear (date) {
+  const i = date.month()
+  const y = date.year()
+  const september = 8
```

Comment:
september is the 9th month - what am i not understanding when reading the code?

Reply:
moment.month() is zero-based, so it will return 0-11 when doing a get, and it expects a value of 0-11 when setting passing a value in.

I have no idea why dates want to be zero-based, as we have considerable mental models of Jan being the 1st month, and not the 0th.

Reply:
i did wonder if it was that.  very odd design choice

Reply:
I've added a comment as a reminder